### PR TITLE
Make receiving methods overridable.

### PR DESCRIPTION
**Pull request by boaks**
Signed-off-by: Achim Kraus <achim.kraus@bosch-si.com>

### CODE REVIEW

File: scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java [hunk 1]
```diff
@@ -585,6 +585,8 @@ public final synchronized void start() throws IOException {
 	 * Re-starts the connector binding to the same IP address and port as
 	 * on the previous start.
 	 * 
+	 * Note: intended for unit tests only!
+	 *
```
Comment (sbernard31) [inline scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: Why is it for unit tests only ?

Comment (boaks) [reply scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: It's package scope and final. So my feeling is, it's only usable for unit test.

Comment (boaks) [reply scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: If you want to change it, feel free.
I would like to include this change in the 2.0.0RC2 tomorrow.
Therefore I merge now this PR.

Comment (sbernard31) [reply scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java]: Uups sorry to not reply about that.

You can merge this !

**Pull request merged**